src/main/java/tool/Storage.java

```diff
@@ -0,0 +1,161 @@
+package tool;
+
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.Todo;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+import java.util.ArrayList;
+
+
+public class Storage {
+    protected String filePath;
+    protected FileWriter fw;
+
+
+    public Storage(String filePath) {
+        try {
+            this.filePath = filePath;
+            fw = new FileWriter(filePath, true);
+        } catch(IOException e) {
+            e.printStackTrace();
+        }
+
+    }
+
+    /**
+     * Loads tasks from the .txt file
+     * @param commands
+     * @return the updated array list of tasks
+     */
+    public ArrayList<Task> load(ArrayList<Task> commands) {
+        try {
+            FileReader fr = new FileReader(this.filePath);
+            BufferedReader br = new BufferedReader(fr);
```

Comment:
`FileReader` and `BufferedReader` can be stored in `Storage` as a field since they are used in most methods.